website/docs/deployment/aws/index.md

```diff
@@ -0,0 +1,79 @@
+- **Security and Compliance**: Deploy Spice.ai within your AWS security perimeter using features like [VPC](https://aws.amazon.com/vpc/) isolation, [security groups](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-security-groups.html), [IAM roles](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html) to meet organizational compliance requirements.
+
+## Deployment Options
+
+### Amazon EKS (Elastic Kubernetes Service)
+
+Leverage [Kubernetes](https://kubernetes.io/) orchestration with [Amazon EKS](https://aws.amazon.com/eks/) for containerized Spice.ai deployments.
+
+1. **Create an EKS Cluster**:
+   - Use the [AWS Management Console](https://console.aws.amazon.com/eks/), [AWS CLI](https://docs.aws.amazon.com/cli/latest/reference/eks/), or [eksctl](https://eksctl.io/) to create your cluster
+   - Configure [node groups](https://docs.aws.amazon.com/eks/latest/userguide/managed-node-groups.html) according to your workload requirements
+   - (Optional) Use [EKS Fargate profiles](https://docs.aws.amazon.com/eks/latest/userguide/fargate.html) for serverless container deployment
+
+2. **Deploy Spice.ai on EKS**:
+   - Apply Spice.ai Kubernetes manifests via [Helm chart](https://spiceai.org/docs/deployment/kubernetes)
+   - Configure persistent storage using [Amazon EBS](https://aws.amazon.com/ebs/) or [Amazon EFS](https://aws.amazon.com/efs/)
+   - Set up ingress with the [AWS Network Load Balancer (NLB)](https://docs.aws.amazon.com/eks/latest/userguide/network-load-balancing.html)
+   - (Optional) Automate cluster and resource provisioning with Infrastructure as Code (IaC) tools such as [AWS CloudFormation](https://aws.amazon.com/cloudformation/) or [Terraform](https://www.terraform.io/) for consistent, repeatable deployments
+
+For comprehensive instructions and advanced configuration options, refer to the [Amazon EKS User Guide](https://docs.aws.amazon.com/eks/latest/userguide/what-is-eks.html), [EKS Best Practices Guide](https://aws.github.io/aws-eks-best-practices/), and [Spice.ai Kubernetes Deployment Guide](https://spiceai.org/docs/deployment/kubernetes).
+
+### EC2 / AWS CloudFormation
+
+Deploy Spice.ai directly on [Amazon EC2](https://aws.amazon.com/ec2/) instances for maximum control over the environment.
+
+1. **Manual EC2 Deployment**:
+   - Launch an [EC2 instance](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EC2_GetStarted.html) with your preferred Linux distribution
+   - Install [Docker](https://docs.docker.com/engine/install/)
+   - Pull the Spice.ai Docker image from [Docker Hub](https://hub.docker.com/r/spiceai/spiceai) and run it on your EC2 instance
+   - (Optional) Use Infrastructure as Code (IaC) tools like [AWS CloudFormation](https://aws.amazon.com/cloudformation/) or [Terraform](https://www.terraform.io/) to automate the provisioning, configuration, and management of EC2 resources for repeatable and consistent deployments
+
+2. **Automated EC2 Deployment with CloudFormation**:
```

Comment:
This must be updated once we have dedicated CloudFormation template for Spice.ai deployment